src/bundle/Resources/public/scss/_custom.scss

```diff
@@ -109,3 +111,22 @@ $link-color: $ez-color-hyperlink;
 
 // Body
 $body-bg: $ez-ground-base-pale;
+
+// Pagination
+
+$pagination-color: $ez-black;
+$pagination-bg: $ez-pagination-bg;
+
+$pagination-active-color: $ez-black !default;
+$pagination-active-bg: $ez-white !default;
+$pagination-active-border-color: $ez-white !default;
+
+$pagination-disabled-color: $ez-white !default;
+$pagination-disabled-bg: $ez-ground-base-medium !default;
+$pagination-disabled-border-color: $ez-ground-base-medium !default;
+
+$pagination-margin: 0.25rem !default;
+$pagination-font-size: 0.9rem !default;
```

Comment:
if the variable is not in bootstrap it should be prefixed with `ez-` and added to the top of file (like you did with `$ez-pagination-bg`). You can skip `!default` as well.